tests/cover/test_conjecture_float_encoding.py

```diff
@@ -109,13 +110,18 @@ def test_floats_round_trip(f):
     assert float_to_int(f) == float_to_int(g)
 
 
-finite_floats = st.floats(allow_infinity=False, allow_nan=False, min_value=0.0)
+finite_floats = st.builds(
```

Comment:
This is only used in the definition of `integral_floats` - I'd just combine them.